chromium_src/chrome/browser/ui/views/download/download_item_view.h

```diff
@@ -6,6 +6,33 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_DOWNLOAD_DOWNLOAD_ITEM_VIEW_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_DOWNLOAD_DOWNLOAD_ITEM_VIEW_H_
 
+// Note: SetMode method name is quite common. To re-define only SetMode in
```

Comment:
maybe it is worth to replace the override with a patch, just to stay on a safe side? @bridiver